app/src/main/java/com/example/henzoshimada/feeltrip/ElasticSearchController.java

```diff
@@ -31,6 +31,8 @@
 
         @Override
         protected Void doInBackground(Mood ... moods ) {
+            if(android.os.Debug.isDebuggerConnected())
```

Comment:
This segment here actually allows us to debug the thread, whereas before it was impossible to set up breakpoints